pwndbg/gdblib/file.py

```diff
@@ -107,7 +107,7 @@ def readlink(path):
 
     if is_qemu:
         if not os.path.exists(path):
-            path = os.path.join(pwndbg.gdblib.qemu.root(), path)
+            path = os.path.join(pwndbg.gdblib.qemu.root() or "", path)
```

Comment:
```suggestion
            # The or "" is needed since .root() may return None
            # Then we just use the path (it can also be absolute too)
            path = os.path.join(pwndbg.gdblib.qemu.root() or "", path)
```